server/src/main/java/org/candlepin/audit/EventBuilder.java

```diff
@@ -76,6 +88,17 @@ public EventBuilder setEventData(Eventful entity) {
                     }
                 }
             }
+            if (event.getTarget().equals(Target.POOL) && event.getType().equals(Type.CREATED)) {
+                Map<String, String> eventData = new HashMap<String, String>();
+                eventData.put("subscriptionId", ((Pool) entity).getSubscriptionId());
+                try {
+                    event.setEventData(mapper.writeValueAsString(eventData));
+                }
+                catch (JsonProcessingException e) {
+                    log.error("Error while building JSON for pool.created event.");
```

Comment:
The logger isn't clear on this, but you can append the exception in the output here by adding it as a parameter without having a matching substitution in the string. I.e.
```
log.error("Error while building JSON for pool.created event", e);
```